lazyflow/operators/ioOperators/opStreamingHdf5Reader.py

```diff
@@ -126,7 +133,14 @@ def execute(self, slot, subindex, roi, result):
         if result.flags.c_contiguous:
             hdf5File[internalPath].read_direct( result[...], key )
         else:
-            result[...] = hdf5File[internalPath][key]
+            if len(hdf5File[internalPath].shape) == 2:
+                roi.start = (roi.start[0], roi.start[1])
```

Comment:
Here we now assume, that it can have more than 2d, so 
`roi.start = roi.start[:-1]  # this excludes the last element, what we expect to be the channel, same for roi.stop